### PR TITLE
Issue#78: Update README of the project

### PR DESCRIPTION
**Pull request by mykytam**
dev
## ZenHub 

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009/issues/ita-social-projects/home/78)

## Code reviewers

- [ ] @alexorlenko 
- [ ] @Arthurpanda 
- [ ] @mykytam 

### Second Level Review

- [ ] @kh0ma 
- [ ] @DzigMS 

## Summary of issue

Update project's README, add additional package in the project for documentation and make .github templates more standard.

## Summary of change

Project description files are added and updated.

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,12 +60,17 @@ It include next modules:
...
+## About the project
+- You can read more about the project's technologies and modules description  in   <a href="https://github.com/ita-social-projects/Home/tree/dev/home-docs" target="_blank">home-docs</a> package.
+
 ## Installation
 
 - All the `code` required to get started
 - Images of what it should look like
 
 ### Required to install
+- Java 11
```
Comment (kh0ma) [inline README.md:72]: Should we add maven to the list of required software?

File: home-docs/starting-the-project.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## What you need to know about the project
+
+### Starting the project locally
+When you have successfully cloned the project, you need to do some steps to start the application.
+Make sure that you have Java 11 installed.
+Firstly, you need create database `home` using PostgreSQL. 
+Then you need to create environmental variables that are defined in `application.properties` file.  
+If you want to fill database with data, you need to activate `dev` profile 
+and use `spring.jpa.hibernate.ddl-auto=create` in `application.properties`.
```
Comment (kh0ma) [inline home-docs/starting-the-project.md]: I don't think that `spring.jpa.hibernate.ddl-auto=create` should be required here. Only migrations have to be a source of our DDL even for local development.

**Review by kh0ma — DISMISSED** [review]

**Review by alexorlenko — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mykytam) [reply README.md:72]: Yes, we should.

File: home-docs/starting-the-project.md (hunk 2 above)
Comment (mykytam) [reply home-docs/starting-the-project.md]: Yes, it makes sense.

**Review by Arthurpanda — APPROVED** [review]

**Pull request merged**